kenya_compliance/kenya_compliance/overrides/server/stock_ledger_entry.py

```diff
@@ -297,7 +313,14 @@ def get_stock_recon_movement_items_details(
     items_list = []
     # current_qty
 
-    for item in records:
+    for i in records:
+        try:
```

Comment:
Once again, the exception handling isn't needed since the properties will be bound at runtime. The properties, valuation_rate and quantity_difference are also provided in the item/i object when iteration begins negating the need to make a db call using get_doc(). The records object already has all the relevant information.